GOVERNANCE.md

```diff
@@ -0,0 +1,106 @@
+# Project Governance
+
+- [Maintainers](#maintainers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [CNCF Resources](#cncf-resources)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Voting](#voting)
+
+## Maintainers
+
+KubeVirt Maintainers govern the project. Maintainers collectively manage the 
+project's resources and contributors, and speak for the project in public.  The
+maintainers collectively decide any questions that cannot be resolved at the 
+individual repository level, and provide strategic guidance for the project
+overall.
+
+The current maintainers can be found in [MAINTAINERS](./MAINTAINERS.md).  
+
+This privilege is granted with some expectation of responsibility: maintainers
+are people who care about the KubeVirt project and want to help it grow and
+improve. A maintainer is not just someone who can make changes, but someone who
+has demonstrated their ability to collaborate with the team, get the most
+knowledgeable people to review code and docs, contribute high-quality code, and
+follow through to fix issues.
+
+A maintainer is a contributor to the project's success and a citizen helping
+the project succeed.
+
+## Selecting Maintainers
+
+The current project maintainers will periodically review contributor activities
+to see if additional project members may be promoted to maintainers. 
+
+For nominations, the maintainers will look at the following criteria:
+
+  * Commitment to the project: have they participated in discussions, 
+    contributions, and reviews for 1 year or more?
+  * Does the person show leadership in one of these areas?
+    * Active approver or reviewer in core or subprojects
+    * SIG leadership
+    * Mentoring other project contributors
+  * Does the candidate bring new perspectives or community connections to the 
+    maintainers?
+  * Do they understand how the project works (policies, processes, etc)?
+  * Are they willing to take on the additional duties of a maintainer?
+
+A candidate must be proposed by an existing maintainer by filing an issue in the
```

Comment:
How about, instead, doing having the proposal as a PR to the MAINTAINERS.md file?

Reply:
that's fine, too